Extension/package.json

```diff
@@ -457,6 +457,17 @@
             ],
             "scope": "resource"
           },
+          "C_Cpp.commentFormatting": {
```

Comment:
Without "markdown" in the setting name, I don't think there's enough context for "subsetEnabled" to be meaningful.

I was going to suggest "markdownInComments" but it looks like @sean-mcmanus already suggested that name, so +1 for that suggestion.

Reply:
Yeah, as stated previously, I don't think formatting should be in the setting name, because then when users search for formatting settings this would appear in the results, and it's not a formatting setting. "markdownInComments" seems fine to me.